media/js/firefox/whatsnew/whatsnew-117-na-relay.es6.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+const relayAnimationWrapper = document.querySelector('.wnp-relay-animation');
+const relayDropdown = document.querySelector('.relay-dropdown');
+const relayInput = document.querySelector('.relay-input');
+const inputText = document.querySelector('.relay-input span');
+
+if (window.Mozilla.Utils.allowsMotion()) {
+    relayAnimationWrapper.addEventListener('animationend', (e) => {
+        if (e.animationName === 'pop-in') {
+            relayAnimationWrapper.classList.add('click-animation');
+        }
+
+        if (e.animationName === 'click' && e.target === relayAnimationWrapper) {
+            relayDropdown.classList.add('active');
+            setTimeout(() => {
+                inputText.textContent = '';
+            }, 500);
+        }
+    });
+
+    relayDropdown.addEventListener(
+        'animationend',
+        (e) => {
+            if (e.animationName === 'button-press') {
+                relayDropdown.style.opacity = 0;
+                relayInput.classList.add('animation-end');
+                inputText.textContent = 'hxty0y40@mozmail.com';
+            }
+        },
+        false
+    );
+} else {
+    inputText.textContent = 'hxty0y40@mozmail.com';
+    document.querySelector('.wnp-content-main').classList.add('static');
```

Comment:
It's an edge case, but if JS fails or is disabled, this doesn't happen and it defaults to the "enter your email" field. Maybe the reduced motion static state could also be the default state if there's no JS?

Though with the traffic cop redirect and if VPN is the default page, it's possible literally nobody would see this page without JS anyway, so maybe it's not worth the effort. Especially if it means completely reworking things. So yeah... up to you if you want to bother with it.

Reply:
OH! I think an easy compromise is if there's no JS just hide the fake field entirely and show the wordmark. That could be done just in CSS with the `.no-js` class and wouldn't have change anything else around.

Reply:
Oh yeah, nice idea!